src/lib.rs

```diff
@@ -161,24 +159,20 @@ pub use crate::error::Error;
 #[allow(dead_code)]
 mod util;
 
-#[cfg(target_os = "vxworks")]
+#[cfg(all(
+    unix,
+    not(any(
+        target_os = "ios",
+        target_os = "fuchsia",
+        target_os = "hermit",
+        target_os = "l4re"
+    ))
+))]
```

Comment:
BTW personally I would prefer to use an explicit enumeration of targets. Size-wise it will not be that bad:
```rust
#[cfg(any(
    target_os = "android", target_os = "dragonfly", target_os = "emscripten",
    target_os = "freebsd", target_os = "haiku",     target_os = "illumos",
    target_os = "linux",   target_os = "macos",     target_os = "netbsd",
    target_os = "openbsd", target_os = "redox",     target_os = "solaris",
    target_os = "vxworks",
))]
#[allow(dead_code)]
mod util_libc;
```

Reply:
I agree, but think this should be fixed in a separate `0.2` PR, I'll open one when this gets merged.

Reply:
Here's the basic idea: https://github.com/josephlr/getrandom/commit/e47483b3a82481d2ed7c544284584708510b08ac